apps/chat-e2e/src/assertions/promptPreviewModalAssertion.ts

```diff
@@ -0,0 +1,66 @@
+import { ElementState, ExpectedMessages } from '@/src/testData';
+import { promptPreviewModal } from '@/src/ui/webElements/promptPreviewModal';
+import { expect } from '@playwright/test';
+
+export class PromptPreviewModalAssertion {
```

Comment:
most of the simple assertions are implemented within `BaseAssertion` class, let's use it as a parent one